Vision Pro Support

durul opened the pull request.

saulamsal commented:
Hey, thanks for the PR, @durul! Will review it soon and see if we can merge it.

saulamsal commented:
I can merge this if you replace styled-components with built-in StyleSheet. The goal is to keep styling unopinionated so users can easily implement their own styling solutions.

durul commented:
> built-in StyleSheet

Hello, I refactored VisionContainer and added a HoverableView component with a built-in StyleSheet. If there is still a requirement, please point out a specific component.

Merry Xmas
